examples/pipelines/commoncrawl/components/load_from_commoncrawl/src/main.py

```diff
@@ -0,0 +1,71 @@
+"""This component loads a dataset from CommonCrawl based on a given index."""
+import logging
+import typing as t
+
+import io
+import boto3
+import gzip
+
+import dask.dataframe as dd
+import pandas as pd
+
+from fondant.component import LoadComponent
+
+logger = logging.getLogger(__name__)
+
+S3_BASE_URL = "s3://commoncrawl/crawl-data"
+S3_COMMONCRAWL_BUCKET = "commoncrawl"
+
+
+def fetch_warc_file_from_s3(s3_bucket: str, s3_key) -> dd.DataFrame:
```

Comment:
Can you add a docstring for the argumens. 

Would also prefer to change `s3_key` to `bucket_path`. Feels more intuitive

Reply:
`s3_key` is the key of the object in the bucket not the bucket path. Would `object_key` be a better name?

Reply:
my bad, just read on it and it seems to be an AWS specific notation (still used to GCP). It's fine to leave it as is

Reply:
ok great!